==========================================
SLURM_CLUSTER_NAME = param-shivay
SLURM_JOB_ACCOUNT = comp_sci_engg
SLURM_JOB_ID = 278096
SLURM_JOB_NAME = roles
SLURM_JOB_NODELIST = gpu003
SLURM_JOB_USER = aman.gupta.cse15.iitbhu
SLURM_JOB_UID = 5180
SLURM_JOB_PARTITION = gpu
SLURM_TASK_PID = 202500
SLURM_SUBMIT_DIR = /home/aman.gupta.cse15.iitbhu/roles
SLURM_CPUS_ON_NODE = 2
SLURM_NTASKS = 1
SLURM_TASK_PID = 202500
==========================================
gpu003
Computing device:  cuda:0
Nodes:  34
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  62
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  198
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  332
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  4039
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  5242
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  1222
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  4941
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  112
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  297
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  1589
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  77
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
Computing device:  cuda:0
Nodes:  115
Computing robustness.

Traceback (most recent call last):
  File "run.py", line 30, in <module>
    rob.calculate()
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 71, in calculate
    self.x_values = numpy.append(self.x_values,self.kfold (edges_new , self.nodes))
  File "/home/aman.gupta.cse15.iitbhu/roles/metrics.py", line 93, in kfold
    feat_mat,role_mat = train(data,self.device,self.method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 59, in train
    encoder = autoencode(dataset,device,20,method)
  File "/home/aman.gupta.cse15.iitbhu/roles/train.py", line 12, in __init__
    self.adj = self.adj.to(device)
RuntimeError: CUDA error: out of memory
